app/jobs/roles/terminate_neuanmeldungen_nv_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2025, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+class Roles::TerminateNeuanmeldungenNvJob < RecurringJob
```

Comment:
```suggestion
class Roles::TerminateStaleNeuanmeldungenNvJob < RecurringJob
```
Vielleicht etwas sprechender? :thinking: